### ISSUE TITLE
Spatialite Cache

### ISSUE THREAD

**Issue by jseppi** [Esri/koop#176]
Looks like there used to be support for Spatialite that has since been removed with the move to plug-ins for cache providers. Will it be coming back as a plug-in as well (or am I not piecing this all together properly?)?

**Comment by chelm** [comment]
Yes it was removed during a refactor and needs to come back as a plugin.
